src/quo2/components/tags/tags.cljs

```diff
@@ -1,7 +1,13 @@
 (ns quo2.components.tags.tags
   (:require [reagent.core :as reagent]
             [quo.react-native :as rn]
-            [quo2.components.tags.tag :as tag]))
+            [oops.core :refer [oget]]
+            [status-im.ui.components.react :as react]
+            [status-im.utils.core :as utils]
+            [quo2.components.tags.tag :as tag]
+            [quo2.components.tabs.tabs :as tabs]))
+
+(def default-tab-size 32)
```

Comment:
Can we make it private to this namespace?
`(def default-tab-size 32)` -> `(def ^:private default-tab-size 32)`

Reply:
cool